.github/workflows/ci-on-pull-request-k3d.yaml

```
@@ -7,6 +7,7 @@ on:
     paths:
       - ".github/workflows/ci-on-pull-request-k3d.y*ml"
       - "defense-unicorns-distro/**"
+      - "values/**"
       - "k3d/values/**"
       - "k3d/zarf-config.y*ml"
       - "k3d/zarf.y*ml"
```

could just make this...
```suggestion
      - "k3d/**"
```
to match the aws pattern